libfuse/file.go

```diff
@@ -11,19 +11,41 @@ import (
 	"golang.org/x/net/context"
 )
 
+type eiCache struct {
+	ei  libkbfs.EntryInfo
+	rid string
+}
+
 // File represents KBFS files.
 type File struct {
 	folder *Folder
 	node   libkbfs.Node
+
+	// for the Attr call after Create
+	eiCache *eiCache
```

Comment:
But the request ID is "guarding" this right? So it means the cached EntryInfo is only valid for this particularly request ID. I'll add a comment for it?